dietpi/dietpi-software

```diff
@@ -3435,6 +3441,87 @@ _EOF_
 
 		fi
 
+		software_id=99 # Prometheus Node Exporter
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+
+			# Use the default port
+			local port="9100"
+
+			local arch=''
+			case $G_HW_ARCH_NAME in
+				'armv6l') arch='armv6';;
+				'armv7l') arch='armv7';;
+				'aarch64') arch='arm64';;
+				'x86_64') arch='amd64';;
+			esac
+
+			# Reinstall: Remove previous instances
+			[[ -e '/usr/local/bin/node_exporter' ]] && G_EXEC rm -R /usr/local/bin/node_exporter
+			[[ -e '/usr/local/bin/raspberrypi_exporter' ]] && G_EXEC rm -R /usr/local/bin/raspberrypi_exporter
+
+			# Download binaries
+			Download_Install $(curl -sSfL 'https://api.github.com/repos/prometheus/node_exporter/releases/latest' | mawk -F\" "/\"browser_download_url\": \".*node_exporter-.*.linux-$arch.tar.gz\"/{print \$4}")
+			G_EXEC mv node_exporter*/node_exporter /usr/local/bin/
+			G_EXEC chmod +x /usr/local/bin/node_exporter
+
+			# User
+			Create_User node_exporter
+
+			# Working directory
+			G_EXEC mkdir -p /var/lib/node_exporter/textfile_collector
+			G_EXEC chown -R node_exporter:node_exporter /var/lib/node_exporter
+
+			# Services
+			cat << _EOF_ > /etc/systemd/system/node_exporter.service
+[Unit]
+Description=Node Exporter
+
+[Service]
+User=node_exporter
+ExecStart=/usr/local/bin/node_exporter --web.listen-address=":$port" --collector.textfile.directory /var/lib/node_exporter/textfile_collector
+
+[Install]
+WantedBy=multi-user.target
+_EOF_
+
+			aSTART_SERVICES+=('node_exporter')
+
```

Comment:
I guess this shall be started by `dietpi-services`? In this case, simply add it to the list there, which makes `aSTART_SERVICES` obsolete here. Same for `raspberrypi_exporter`.
```suggestion
```

Reply:
After removing this, the services aren't started automatically after install. What's the right way of doing this without `aSTART_SERVICES`?